docker-compose/technical/prometheus.yml

```
@@ -0,0 +1,12 @@
+global:
+  scrape_interval: 2s
+  evaluation_interval: 2s
+
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['172.17.0.1:8000']
+  - job_name: 'apps'
+    metrics_path: '/actuator/prometheus'
+    static_configs:
+      - targets: ['172.17.0.1:5008']
```

call the container directly by using network container name, so we haven't to manage the IP, nor have to pass by host port
```suggestion
      - targets: ['loadflow-server:80']

```